internal/users/users.go

```diff
@@ -557,3 +549,49 @@ func DeleteUserSession(token string) error {
 
 	return nil
 }
+func UserValidVal(userValid *UserValid) error {
```

Comment:
what does `UserValidVal` mean? I can't get the slightest idea what it does based on the name